fearless_simd_gen/src/mk_sse4_2.rs

```diff
@@ -83,18 +76,18 @@ fn mk_simd_impl() -> TokenStream {
     let mut methods = vec![];
     for vec_ty in SIMD_TYPES {
         for (method, sig) in ops_for_type(vec_ty, true) {
-            let b1 = (vec_ty.n_bits() > 128 && !matches!(method, "split" | "narrow"))
+            let too_wide = (vec_ty.n_bits() > 128 && !matches!(method, "split" | "narrow"))
                 || vec_ty.n_bits() > 256;
 
-            let b2 = !matches!(method, "load_interleaved_128")
+            let acceptable_wide_op = !matches!(method, "load_interleaved_128")
                 && !matches!(method, "store_interleaved_128");
 
-            if b1 && b2 {
+            if too_wide && acceptable_wide_op {
```

Comment:
The variable named `acceptable_wide_op` seems to have opposite meanings in this and the avx2 file. Pointing this out as a point of suspicion, rather than something which I've confirmed is wrong.

Reply:
I'd renamed the variables here for consistency with the AVX2 version, but forgot to invert the conditions. Fixed that now.